src/components/AddBeer.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+export class AddBeer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      tagline: '',
+      description: '',
+      first_brewed: '',
+      brewers_tips: '',
+      attenuation_level: 0,
+      contributed_by: '',
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({ ...this.state, [event.target.name]: event.target.value });
```

Comment:
`handleChange = ({target}) => this.setState(state => ...)`

Si vas a hacer uso del state anterior, have el setState con callback